pages/index.vue

```diff
@@ -85,27 +85,54 @@
         }"
       >
         <legend class="text-2xl sr-only">Your scroll</legend>
-        <div
-          class="grid gap-6 pr-2"
-          :style="{
-            gridTemplateColumns: `repeat(auto-fill, minmax(17rem, 1fr))`,
-          }"
-        >
-          <ScrollPanel
-            v-for="(dragon, i) in scroll.dragons"
-            :key="dragon.id"
-            v-model="scroll.dragons[i]"
-            :settings="userSettings"
-            :recently-added
-            @click="
-              () => {
-                if (!isProcessing) {
-                  dragon.in_garden = !dragon.in_garden;
+        <template v-if="hatchlings.length">
+          <p>Hatchlings</p>
```

Comment:
think the text as well should be smaller here. just concerned with it becoming a lengthy page